pyrasterframes/src/main/python/docs/raster-io.pymd

```diff
@@ -60,10 +60,10 @@ print(tile)
 Generally we'll want to work with multiple bands. If you have several singleband image files of the same scene, `rastersource` can make use of a CSV (or other DataFrame) specifying the band columns and scene rows.
 
 ```python
-csv = f"""
+csv = """
 red,green,blue
-{s2_band(4)},{s2_band(3)},{s2_band(2)}
-"""
+{},{},{}
+""".format(s2_band(4), s2_band(3), s2_band(2))
```

Comment:
yikes. maybe as below ?

```python
csv = "red,green,blue\n" + ','.join([s2_band(b) for b in [4, 3, 2]])

```

Reply:
I'm sure you or someone will hack all that away at some point. Just wanted to show a new modality.